ebml/EbmlCrc32.h

```diff
@@ -68,7 +68,7 @@ DECLARE_EBML_BINARY(EbmlCrc32)
     /*!
       Calls Update() and Finalize(), use to create a CRC32 in one go
     */
-    void FillCRC32(const binary *input, uint32 length);
+    void FillCRC32(const binary *s, uint32 n);
```

Comment:
I think the names should be changed in the implementation. These names are better than `s` and `n`.